src/resources/dashboard.yml

```diff
@@ -2,8 +2,8 @@ dashboard:
   status-messages:
     incomplete:
       message_type: warning
-      title: Your application is incomplete &nbsp; &#128533; # Awkward face emoji
-      subline: Finish it off when you're ready by completing everything below
+      title: Your application is incomplete.
```

Comment:
If we're adding full stops to titles, we should do this to every title for consistency.

Reply:
This should now be fixed.